### PR TITLE
The roles were moved in create user, it is displayed in view and can be edited

### PR DESCRIPTION
**Pull request by fugas20**
The role icon was removed from the user and was placed when creating a new user, in the same way it is displayed when viewing the role and you can also change the role from the edit icon

### CODE REVIEW

File: app/Http/Controllers/UserController.php [hunk 1]
```diff
@@ -11,16 +11,11 @@ class UserController extends Controller
...
-        $users = trim($request->get('text'));
-        $users = DB::table('users')
-            ->select('id', 'name', 'last_name', 'phone', 'email', 'password')
-            ->where('id', 'LIKE', '%' . $users . '%')
-            ->orWhere('name', 'LIKE', '%' . $users . '%')
-            ->orderBy('name', 'asc')
-            ->paginate(10);
-        $user = User::all();
-
-        return view('users.index', compact('users'));
```
Comment (And-Y21) [inline app/Http/Controllers/UserController.php]: creo que solo era necesario agregar esto

       $roles = Role::all();
        $users = User::all();
        return view('users.index', compact('users', 'roles'));

File: app/Http/Controllers/UserController.php [hunk 2]
```diff
@@ -11,16 +11,11 @@ class UserController extends Controller
...
+
+        $roles = Role::all();
+        $users = User::all();
+        return view('users.index', compact('users', 'roles'));
+        
     }
```
Comment (And-Y21) [inline app/Http/Controllers/UserController.php]: checar identado

File: config/app.php [hunk 3]
```diff
@@ -52,7 +52,7 @@
     |
     */
```
Comment (And-Y21) [inline config/app.php]: verificar si esto no rompe el proyecto

File: app/Http/Controllers/UserController.php (hunk 1 above)
Comment (fugas20) [reply app/Http/Controllers/UserController.php]: pero eso es lo que se elimino

**Pull request merged**